services/gitlab/gitlab-issues.service.js

```diff
@@ -0,0 +1,446 @@
+import Joi from 'joi'
+import { optionalUrl } from '../validators.js'
+import { metric } from '../text-formatters.js'
+import GitLabBase from './gitlab-base.js'
+
+const schema = Joi.object({
+  statistics: Joi.object({
+    counts: Joi.object({
+      all: Joi.number().required(),
+      closed: Joi.number().required(),
+      opened: Joi.number().required(),
+    }).required(),
+  }).allow(null),
+}).required()
+
+const queryParamSchema = Joi.object({
+  labels: Joi.string(),
+  gitlab_url: optionalUrl,
+}).required()
+
+const documentation = `
+<p>
+  You may use your GitLab Project Id (e.g. 278964) or your Project Path (e.g. gitlab-org/gitlab )
+</p>
+`
+
+const customDocumentation = `
+<p>
+  Note that only internet-accessible GitLab instances are supported, for example https://jihulab.com, https://gitlab.gnome.org, or https://gitlab.com/.
+  You may use your GitLab Project Id (e.g. 13953) or your Project Path (e.g. gitlab-cn/gitlab ) in <a href="https://jihulab.com">https://jihulab.com</a>
+</p>
+`
+
+const labelDocumentation = `
+<p>
+  Support Multi-label and <a href="https://docs.gitlab.com/ee/user/project/labels.html#scoped-labels">Scoped labels</a>, Multi-lable please use <code>,</code> to separate.
+</p>
+`
+
+export default class GitlabIssues extends GitLabBase {
+  static category = 'issue-tracking'
+
+  static route = {
+    base: 'gitlab/v/issues',
```

Comment:
We shouldn't have the `v` route member, which is only applicable for version-related badges

```suggestion
    base: 'gitlab/issues',
```